docs/source/support/release-notes.rst

```diff
@@ -6,6 +6,17 @@ Release notes
 
 This page contains release notes for the SDK.
 
+DAML-LF
```

Comment:
a few things:

- this should be in unreleased.md, I think?
- for breaking changes we usually add something like `[BREAKING]` before it, though I don't know if you consider it breaking yet?
- can you update this in the contract keys doc, specifically `https://github.com/digital-asset/daml/blob/master/docs/source/daml/reference/contract-keys.rst#fetchbykey` and `https://github.com/digital-asset/daml/blob/master/docs/source/daml/reference/updates.rst#fetchbykey`? I'm aware the restriction won't be default when you merge it, but I'd rather get them updated now rather than having it forgotten later.

Reply:
* Added to unreleased
* It's not a breaking change yet, I explain why in the note
* Why? It's not active yet. It'd just be wrong.

Reply:
given it's an extra restriction, it doesn't hurt to add it now - following the restriction will work, and getting people used to this earlier seems sensible. It may be incorrect that this is the restriction, but behaving as though it's correct still works fine for users.

And I think it's pretty likely that if you don't update it now, it won't happen when the switch to 1.6 happens, but I could be wrong.

Reply:
OK, I've added a warning about the future behavior of these functions in the docs.